js/app/cuttingTheMustard.js

```diff
@@ -12,9 +12,9 @@ Modernizr.load({
 });
 //set conditional assets for main.js
 var globalSiteSpecificVars = {
-	pathToJquery: '//cdn.ucl.ac.uk/indigo/js/lib/jquery-1.9.1.min'
-	,googleAnalyticsIdsArray: []//specify array of site specific id's NOT UCL generic UA-943297-1
+//	pathToJquery: '//cdn.ucl.ac.uk/indigo/js/lib/jquery-1.9.1.min',
```

Comment:
Main.js has `jquery: globalSiteSpecificVars.pathToJquery`. Which sounds to me like it's expecting PathToQuery to have a value. Although where that is then used I don't know...!
e.g. `https://cdn.ucl.ac.uk/skins/UCLDrupalIndigoSkin/default-theme/main.js`

Reply:
I'll have a look...

Reply:
New version to look at using the Drupal JS version.